streams/src/main/java/org/apache/kafka/streams/processor/internals/assignment/AssignorConfiguration.java

```diff
@@ -38,6 +38,9 @@
 import static org.apache.kafka.streams.processor.internals.assignment.StreamsAssignmentProtocolVersions.LATEST_SUPPORTED_VERSION;
 
 public final class AssignorConfiguration {
+    public static final String HIGH_AVAILABILITY_FLAG_CONFIG = "high.availability.enabled";
```

Comment:
```suggestion
    public static final String HIGH_AVAILABILITY_FLAG_CONFIG = "internal.high.availability.enabled";
```
To document that it's undocumented ;)